Documentation/General/finalReport/InteractionDesign.tex

```diff
@@ -1,2 +1,106 @@
+\item User selects the appropriate buttons for valence and arousal
+\item User clicks submit
+\item User gets 6 points
+\item User comes back to the menu to click a new song to classify
+\end{itemize}
+
+
+%Beschrijving van context inquiry
+\section{Context inquiry}
+% A contextual inquiry interview is usually structured as an approximately two-hour, one-on-one interaction in which the researcher watches the user do their normal activities and discusses what they see with the user.
+We did a context inquiry with 5 other students a the tu delft.
+All 5 of us watched a different person, but we were sitting in the same room.
+They were separated by some free space, so they would not be influenced by each other.
+We just asked them to play with the system and we would see how they do.
+To see how they experienced the system is quite interesting, a user acts different in the first 5 minutes of system use.
+Then the system is new and such, but later on they of course knew all the features.
+These are two very important aspects.
+First off all we want to bind the user to our system, so he will use it and then we want them to keep using the system.
+So, that the user comes back often, because he likes the system.\\
+They could find almost all the features, they immediately found the rating of the song and the classify game.
+Only the leaderboard, where you come by clicking at the points was a bit too hidden away.
+This is something minor, but we want to change this in the future.
+
+%More about context inquiry
+
+%Beschrijving van product design, claim analyze over gebruik ondersteuned met HCI literatuur
+\section{Product design and claim analysis}
+MoodCat is developed with the end-user as starting point, we tried to make the interface as intuitive as possible.
+The interface should not contain a lot of information, so that the user remains the overview.
+%What to say more
+
+\subsection{Claim analysis}
+MoodCat is developed, so that people can listen to music and chat with people who are in the same mood.
+We have resourced this and gamers streaming on twitch is very popular, due to the twitch chat.
+A record company Monstercat streams music on twitch, this is very popular and people seem to enjoy this.
+We see this coming back in the interaction session with possible end-users, that they really enjoy the chat.
+
+
+%Beschrijving van een bruikbaarheidsevaluatie uitgevoerd met het product
+\section{Usability evaluation}
+\subsection{Cognitive Walkthroughs}
+During the design of the product we did multiple usability evaluations.
+We have done quite a few Cognitive Walkthroughs, with changing possible end-users.
+We done this with the thinking out loud protocol, so we always knew there they where stuck or did not fully understand something.
+When you do not ask people to think out loud, they would sometimes hide some mistakes, which later on can be very important.
+We gave the users the task to explore the functionalities of the system.
```

Comment:
I think we lack a detailed report on what actually were their experiences, what was the feedback that they came with, and did they use the system in ways we would have expected.